streams/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/transform/MergeTransformRuntime.java

```diff
@@ -66,7 +66,12 @@ public String toString() {
 
         @Override
         public TransformRuntime create(Transform transform) {
-            return new MergeTransformRuntime((MergeTransform) transform);
+            if (transform instanceof MergeTransform) {
```

Comment:
This pattern is used from so many places. Could we extract a method for this pattern?